org-roam-compat.el

```diff
@@ -80,6 +80,9 @@
 (define-obsolete-function-alias 'org-roam-db--maybe-update 'org-roam-db--update-maybe
   "org-roam 1.1.0")
 
+(when (version< (org-version) "9.3")
+  (fset 'org-link-make-string 'org-make-link-string))
```

Comment:
```suggestion
(when (version< (org-version) "9.3")
  (defalias 'org-make-link-string 'org-link-make-string nil))
```
defalias is the proper choice here because the user will be able to see the aliasing in the functions' help entries.

Reply:
Okay. So long as you take care of the `package-lint` error. You could just exclude the whole file,

Reply:
@jethrokuan: 
makem.sh does have an --exclude option which would allow us to circumvent the CI for cases such as these, but I don't foresee us needing these types of changes too often. I think it's wiser to just manually review/accept the changes than turn off linting for `org-roam-compat.el`.

Thoughts?

Reply:
Perhaps I am misunderstanding. Won't this change cause CI to fail for all future PR's as well? I assume that `pacakge-lint` is run on the whole repo.

Reply:
I think it's alright to turn off linting for `org-roam-compat.el` actually.